src/cmap/commands.ts

```diff
@@ -281,7 +281,6 @@ export interface MessageHeader {
 
 /** @internal */
 export interface OpResponseOptions extends BSONSerializeOptions {
-  raw?: boolean;
```

Comment:
Just adding a note for these from virtual office. These are removed as they are present in the options superclass.